legacy_app.py

```
@@ -83,7 +83,7 @@
 from core.bmi.compat_plan import legacy_plan_category
 from core.bmi.engine import _normalize_bool_flag
 from bmi_visualization import MATPLOTLIB_AVAILABLE, generate_bmi_visualization
-from core.fingerprint_security import compute_fingerprint
+from core.fingerprint_security import _client_fingerprint, compute_fingerprint
```

Flake8 (7.3.0)

[error] 'core.fingerprint_security.compute_fingerprint' imported but unused

(F401)